google-cloud-workflows-bom/pom.xml

```diff
@@ -64,21 +64,30 @@
 
   <dependencyManagement>
     <dependencies>
-
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-workflows</artifactId>
         <version>0.3.2-SNAPSHOT</version><!-- {x-version-update:google-cloud-workflows:current} -->
       </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>grpc-google-cloud-workflows-v1beta</artifactId>
+        <version>0.3.2-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-workflows-v1beta:current} -->
+      </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>grpc-google-cloud-workflows-v1</artifactId>
+        <version>0.3.2-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-workflows-v1:current} -->
+      </dependency>
       <dependency>
         <groupId>com.google.api.grpc</groupId>
         <artifactId>proto-google-cloud-workflows-v1beta</artifactId>
         <version>0.3.2-SNAPSHOT</version><!-- {x-version-update:proto-google-cloud-workflows-v1beta:current} -->
       </dependency>
       <dependency>
         <groupId>com.google.api.grpc</groupId>
-        <artifactId>grpc-google-cloud-workflows-v1beta</artifactId>
-        <version>0.3.2-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-workflows-v1beta:current} -->
+        <artifactId>proto-google-cloud-workflows-v1</artifactId>
+        <version>0.3.2-SNAPSHOT</version><!-- {x-version-update:proto-google-cloud-workflows-v1:current} -->
```

Comment:
same here?

Reply:
Releasing as pre major for now, in anticipation of possible breaking changes.
Will promote to 1.0.0 in 3 weeks.